org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/map/LiteMapPane.java

```diff
@@ -58,10 +59,26 @@
 	// be 3857?)
 	private static final String DATA_PROJECTION = "EPSG:4326";
 
+	// Maximum are allowed in the map
+	public static double MAX_MAP_AREA = 2.0E15;
+
+	public static double MAX_WORLD_AREA = 50000.0;
+
 	private static Color getNewColor(final double alpha) {
 		return new Color((int) (255 - alpha * 171), (int) (255 - alpha * 119), (int) (255 - alpha * 57));
 	}
 
+	public static boolean isViewportAcceptable(final JMapPane map)
```

Comment:
@saulhidalgoaular  - I know that in D-Lite we've got quite a lot of static methods and fields. As we move forward, I'd like us to minimise/reduce their usage.  I _like_ the use of a static method to demonstrate it's a pure function that makes no reference/changes to class fields.  But, I _dislike_ the use of static methods that force the presence of static fields.  But, that's for the future :-)